tags-transformer/README.md

```diff
@@ -0,0 +1,25 @@
+# Rollbar PHP Examples: LaunchDarklyTransformer
+
+This project is an example of how to set up a custom transformer for Rollbar
+reporting for your PHP application. We use this functionality to move the extra
+data sent with the payload to the root level under the `tags` object.
+
+## Installation
+
+1. `cp config/secret.example.php config/secret.php`
+2. Fill out `config/secret.php` with your Rollbar server-side access token.
```

Comment:
From a security standpoint, it's best practice to avoid storing any secrets in the filesystem. I'd suggest removing `config/secret.example.com`, the entry for `config/secret.php` in the `.gitignore` file, and these secret installation instructions. Instead, down below, I'd get the secret from environment variables. See below for further thoughts on this.